Lib.Web.Mvc/JQuery/JqGrid/DataAnnotations/JqGridColumnSearchableAttribute.cs

```diff
@@ -47,7 +46,55 @@ public JqGridSearchOperators SearchOperators
             set { SearchOptions.SearchOperators = value; }
         }
 
-        private JqGridColumnSearchOptions SearchOptions
+				/// <summary>
+				/// Available search operators for the column with the ability to specify their order.
+				/// (default SearchOp.Eq).
+				/// </summary>
+				public string SearchOrderedOperators { 
```

Comment:
This creates second property for representing the same information (SearchOperators) and results in incosistency within the API (one is set by enums while the other takes string, there is a hidden information about which takes precendence etc.).